h/views.py

```diff
@@ -133,5 +189,6 @@ def includeme(config):
     config.add_route('help', '/docs/help')
     config.add_route('onboarding', '/welcome')
     config.add_route('stream', '/stream')
+    config.add_route('atom_stream', '/stream.atom')
```

Comment:
I'm such a nit-picker, so feel free to ignore me, but I think `stream_atom`. I could see having `stream_as2` or `stream_rss`, etc, and the symmetry between the name and the `path.ext` is nice.